proxy-node-shared/src/main/java/uk/gov/ida/notification/session/storage/RedisStorageService.java

```diff
@@ -99,7 +99,7 @@ public Long updateWithVersion(long version, String context, String key, String v
             return null;
         } else if (storedVersion != version) {
             redis.unwatch();
-            throw new VersionMismatchException(String.format("Expected version to be %l but it was %l", version, storedVersion));
+            throw new VersionMismatchException(String.format("Expected version to be %d but it was %d", version, storedVersion));
```

Comment:
I can't find a reference to the "%l" format specifier in Java anywhere. In C it's for a long. Was this just a mistake?

Reply:
I ran the IntelliJ analyse thing and it spotted it - I think whoever did it, guessed it was *long* (maybe from C experience) but that as you say, isn't a thing in Java.  I'm pretty sure I've done it myself in the past but spotted it because I ran the code.  All that would happen is that if that exception was thrown it would output the literal string with `%l` in it instead of the substituted variable.  I would be surprised if that exception has ever been thrown so nobody (except IntelliJ) has ever noticed.